books/dev-books/Database/database.md

```diff
@@ -7,7 +7,13 @@ Please kindly follow this structure when you are contributing
 // List everything in alphabetical order (A - Z)
 // List them as a direct link to the resource (No redirects)
 
+- [Title of the book pdf](direct link to the book)
+
 ### P
 
 - [PostgreSQL](https://goalkicker.com/PostgreSQLBook/)
 - [Practical SQL: A beginners Guide](https://drive.google.com/file/d/1LGcFIlyeYUz0zxghNHlj9KQ7azCGX90G/view?usp=share_link)
+
+### M
+
+- [MongoDB Notes for Professionals](https://goalkicker.com/MongoDBBook/)
```

Comment:
Please adjust the alphabetical order by moving the **###M** section above the **###P** section

Reply:
Oh, my bad.
fixing the problem now.